python/cudf/cudf/core/_base_index.py

```diff
@@ -540,132 +516,12 @@ def fillna(self, value, downcast=None):
 
         return super().fillna(value=value)
 
-    def to_frame(self, index=True, name=None):
```

Comment:
Same as `values`, this should be an abstract method since all subclasses need an implementation.